cuqi/experimental/mcmc/_laplace_approximation.py

```diff
@@ -0,0 +1,150 @@
+import scipy as sp
+import numpy as np
+import cuqi
+from cuqi.solver import CGLS
+from cuqi.experimental.mcmc import SamplerNew
+
+class UGLANew(SamplerNew):
+    """ Unadjusted (Gaussian) Laplace Approximation sampler
+    
+    Samples an approximate posterior where the prior is approximated
+    by a Gaussian distribution. The likelihood must be Gaussian.
+
+    Currently only works for LMRF priors.
+
+    The inner solver is Conjugate Gradient Least Squares (CGLS) solver.
+
+    For more details see: Uribe, Felipe, et al. "A hybrid Gibbs sampler for edge-preserving 
+    tomographic reconstruction with uncertain view angles." arXiv preprint arXiv:2104.06919 (2021).
+
+    Parameters
+    ----------
+    target : `cuqi.distribution.Posterior`
+        The target posterior distribution to sample.
+
+    initial_point : ndarray
+        Initial parameters. *Optional*
+
+    maxit : int
+        Maximum number of inner iterations for solver when generating one sample.
+
+    tol : float
+        Tolerance for inner solver. Will stop before maxit if the inner solvers convergence check reaches tol.
+
+    beta : float
+        Smoothing parameter for the Gaussian approximation of the Laplace distribution. Larger beta is easier to sample but is a worse approximation.
```

Comment:
Could we help the user by specifying a  range or typical values? Less than 1? 1.0e12?

Reply:
Done after group discussion